### PR TITLE
Added missing tests for storing response to authenticated request

### PR DESCRIPTION
**Pull request by akrambek**

### CODE REVIEW

File: src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7234/explicitly.smaxage.and.authorization/connect/server.rpt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+connected
+
+read closed
+
+write nukleus:begin.ext ${http:header(":status", "200")}
+                        ${http:header("content-length", "11")}
+                        ${http:header("date", date)}
+                        ${http:header("last-modified", date)}
+                        ${http:header("cache-control", "s-maxage=15")}
+write "hello world"
```
Comment (dpwspoon) [inline src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7234/explicitly.smaxage.and.authorization/connect/server.rpt]: LGTM, but the cache will inject (I believe) a generated weak etag, if one is not provided in the response.  I would suggest providing one

Comment (dpwspoon) [reply src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7234/explicitly.smaxage.and.authorization/connect/server.rpt]: et all

**Review by dpwspoon — CHANGES_REQUESTED** [review]
Minor feedback

File: src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7234/explicitly.smaxage.and.authorization/connect/server.rpt (hunk 1 above)
Comment (akrambek) [reply src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7234/explicitly.smaxage.and.authorization/connect/server.rpt]: Done.

**Comment by akrambek** [comment]
Updated PR.

**Review by dpwspoon — APPROVED** [review]

**Review by jfallows — APPROVED** [review]

**Pull request merged**